portal/templates/flask_user/second_factor.html

```diff
@@ -1,16 +1,22 @@
 {% extends "layout.html" %}
 {% block main %}
 {% from "flask_user/_macros.html" import render_field %}
-
-    <h3 class="tnth-headline">{{ _("Two Factor Athentication") }}</h3><br />
-
-    <p>{{ _("You should receive an email with a token. Enter the token below for access.") }}</p>
-
-    <form action="{{ url_for('auth.two_factor_auth') }}" method="post">
-        {{ form.hidden_tag() }}
-        {{ render_field(form.key, label=_("Token"), tabindex=5) }}
-
-        <button type="submit" class="btn btn-tnth-primary">{{ _("Validate Token") }}</button>
-    </form>
-
+    <h3 class="tnth-headline">{{ _("Two Factor Athentication") }}</h3>
+    <p>{{ _("You should receive an email with a 4 digit code. Enter the code below for access.") }}</p>
```

Comment:
Just in case we flip-flop a few more times on the number of characters :) 
```suggestion
    <p>{{ _("You should receive an email with a %d digit code. Enter the code below for access.", 4) }}</p>
```

Reply:
agreed.  looks like this already got fixed?